src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -185,7 +185,11 @@ export default class AircraftInstanceModel {
 
         // TODO: There are better ways to ensure the autopilot is on for aircraft spawning inflight...
         if (options.category === FLIGHT_CATEGORY.ARRIVAL) {
-            const bottomAltitude = this.fms.getBottomAltitude();
+            let bottomAltitude = this.fms.getBottomAltitude();
+
+            if (bottomAltitude === Infinity) {
```

Comment:
I agree, but we don't have access to the aircraft's altitude in that scope. Unless you think we should just pass that? :grimacing: